src/ui/zcl_abapgit_gui_repo_over.clas.abap

```diff
@@ -410,55 +436,6 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_OVER IMPLEMENTATION.
   ENDMETHOD.
 
 
-  METHOD zif_abapgit_gui_event_handler~on_event.
```

Comment:
Mmmm... I would leave this here. Point is that with new event_handler registration add any component into the event pipe `gui_services( )->get_hotkeys_ctl( )->register_event_handler( me )`. And it makes sense to process events of the component inside the component. Also there would be no need to expose additional public methods.